sources/platform/actors/development/programming_interface/system_events.md

```diff
@@ -5,25 +5,40 @@ slug: /actors/development/programming-interface/system-events
 sidebar_position: 4
 ---
 
+# System events in Apify Actors
+
 **Learn about system events sent to your Actor and how to benefit from them.**
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ---
 
-The system notifies Actors about various events such as a migration to another server, [abort operation](#abort-another-actor) triggered by another Actor, or the CPU being overloaded.
```

Comment:
Why remove that link?

Reply:
This is dead link not pointing to anything